main.tf

```
@@ -203,6 +203,10 @@ data "aws_region" "this" {}
 data "aws_caller_identity" "this" {}
 
 locals {
+  // Config aggregation isn't enabled for ap-northeast-3
+  // https://docs.aws.amazon.com/config/latest/developerguide/aggregate-data.html
+  is_enabled = module.this.enabled && data.aws_region.this.name != "ap-northeast-3"
```

It would be nice to setup a list of disabled regions e.g. `var.disabled_regions` in a variable that could be overwritten once AWS Config is available in the above region.

---

Yes, and we could do something like 
```suggestion
  is_enabled = module.this.enabled && contains(var.disabled_regions, data.aws_region.this.name)
```

---

4cfb6e1ca0457814f4639e4ff06d830c0f707208